build_darwin.sh

```diff
@@ -1,15 +1,47 @@
 #!/bin/sh
 set -eu
 
-curl -fsSL "https://www.torproject.org/dist/tor-0.3.2.10.tar.gz" -o tor-0.3.2.10.tar.gz
-curl -fsSL "https://www.torproject.org/dist/tor-0.3.2.10.tar.gz.asc" -o tor-0.3.2.10.tar.gz.asc
+curl -fsSL "https://www.openssl.org/source/openssl-$OPENSSL_VERSION.tar.gz" -o openssl-$OPENSSL_VERSION.tar.gz && \
+echo "8c6ff15ec6b319b50788f42c7abc2890c08ba5a1cdcd3810eb9092deada37b0f  openssl-$OPENSSL_VERSION.tar.gz" | shasum -a 256 -c - && \
+tar -xvzf openssl-$OPENSSL_VERSION.tar.gz && \
+cd openssl-$OPENSSL_VERSION && \
+./config --prefix=$PWD/install no-shared no-dso && \
+make
+
+cd ..
+
+curl -fsSL "https://github.com/libevent/libevent/releases/download/release-$LIBEVENT_VERSION/libevent-$LIBEVENT_VERSION.tar.gz" -o libevent-$LIBEVENT_VERSION.tar.gz && \
+curl -fsSL "https://github.com/libevent/libevent/releases/download/release-$LIBEVENT_VERSION/libevent-$LIBEVENT_VERSION.tar.gz.asc" -o libevent-$LIBEVENT_VERSION.tar.gz.asc && \
+
+gpg --keyserver pgp.mit.edu/ --recv B86086848EF8686D && \
+gpg libevent-$LIBEVENT_VERSION.tar.gz.asc && \
+echo "965cc5a8bb46ce4199a47e9b2c9e1cae3b137e8356ffdad6d94d3b9069b71dc2  libevent-$LIBEVENT_VERSION.tar.gz" | shasum -a 256 -c - && \
+tar -zxvf libevent-$LIBEVENT_VERSION.tar.gz && \
+cd libevent-$LIBEVENT_VERSION && \
+./configure \
+            LDFLAGS="-L$PWD/../openssl-$OPENSSL_VERSION/" \
+            CPPFLAGS="-I$PWD/../openssl-$OPENSSL_VERSION/include" \
+            --prefix=$PWD/install \
+            --disable-shared \
+            --enable-static \
+            --with-pic && \
+make && make check
+
+cd ..
+
+curl -fsSL "https://www.torproject.org/dist/tor-$TOR_VERSION.tar.gz" -o tor-$TOR_VERSION.tar.gz
+curl -fsSL "https://www.torproject.org/dist/tor-$TOR_VERSION.tar.gz.asc" -o tor-$TOR_VERSION.tar.gz.asc
 
 gpg --keyserver pgp.mit.edu/ --recv FE43009C4607B1FB
-gpg tor-0.3.2.10.tar.gz.asc
+gpg tor-$TOR_VERSION.tar.gz.asc
 
-tar -xvzf tor-0.3.2.10.tar.gz
-cd tor-0.3.2.10
-./configure && make && make install
+tar -xvzf tor-$TOR_VERSION.tar.gz
+cd tor-$TOR_VERSION && \
+./configure --enable-static-libevent \
```

Comment:
Why not just `--enable-static-tor`? Also add `--disable-asciidoc` cause we don't need that.

Reply:
it enables `--enable-static-zlib` as well. I'll add `--disable-asciidoc`

Reply:
got it.